<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainViewClass</class>
 <widget class="QMainWindow" name="MainViewClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainView</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QPushButton" name="btnMap">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>45</height>
         </size>
        </property>
        <property name="text">
         <string>Mapa</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnTree">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>45</height>
         </size>
        </property>
        <property name="text">
         <string>Arbol</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSimulation">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>45</height>
         </size>
        </property>
        <property name="text">
         <string>Simulacion</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnReports">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>45</height>
         </size>
        </property>
        <property name="text">
         <string>Reportes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLogOut">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>45</height>
         </size>
        </property>
        <property name="text">
         <string>Cerrar Seccion</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedViews">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_5"/>
      <widget class="QWidget" name="page_6"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
